petal/commands/public.py

```diff
@@ -927,6 +928,291 @@ async def cmd_dsmsg(
             proc.stdout.decode()
         )
 
+    async def cmd_lazymath(self, args, src: Src, **_):
+        """Converter for all your basic unit conversions
+
+        Syntax: `{p}lazymath [number]`
+        """
+        if not args:
+            return "Give me numbers. Try again (or edit your message) to have a number"
```

Comment:
Raising an Exception is required to start listening for an edit. Details of which Exceptions allow edits are in [the Exceptions module](https://github.com/hdmifish/petal/blob/master/petal/exceptions.py).

```suggestion
            raise CommandArgsError("Give me numbers. Try again (or edit your message) to have a number")
```